sympy/vector/scalar.py

```diff
@@ -12,6 +13,9 @@ class BaseScalar(Symbol):
     """
 
     def __new__(cls, name, index, system, pretty_str, latex_str):
+        name = str(name)
+        pretty_str = str(pretty_str)
+        latex_str = str(latex_str)
```

Comment:
I'm going to undo this change in my merge because I think it's the wrong thing in Python 2 (since pretty_str could be unicode).